presto-docs/src/main/sphinx/installation/cli.rst

```diff
@@ -14,7 +14,90 @@ make it executable with ``chmod +x``, then run it:
 
     ./presto --server localhost:8080 --catalog hive --schema default
 
-Run the CLI with the ``--help`` option to see the available options.
+The available options for the CLI are:
+
+.. code-block:: none
+
+        --catalog <catalog>
+            Default catalog to connect to
+
+        --client-request-timeout <client request timeout>
+            Client request timeout (default: 2m)
+
+        --debug
+            Enable debug information
+
+        --enable-authentication
+            Enable client authentication
+
+        --execute <execute>
+            Execute specified statements and exit
+
+        -f <file>, --file <file>
+            Execute statements from file and exit
+
+        -h, --help
+            Display help information
+
+        --keystore-password <keystore password>
+            The password for the keystore. This must match the password specified when creating the keystore.
```

Comment:
I think most doc pages in Presto wrap at 80 characters.

Also all descriptions except for this one and a couple of others do not end with a period.